### PR TITLE
Issues Resolved

### PR DESCRIPTION
**Pull request by cq-mohtashim-ali**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -3,37 +3,48 @@ const open = require("open");
...
   ws.on("message", (message) => {
     const data = JSON.parse(message);
-    console.log(data.name);
     userName = data.name;
-    const numbr = check.checkUser(data.name);
-    console.log(numbr);
-    if (numbr === 1) {
+    const isPresent = checkUser(data.name);
+    if (isPresent === true) {
+      const messageObject = {
```
Comment (cq-ubaid-khan) [inline src/app.js:36]: try to simplify things, you can give more descriptive name to checkUser which can remove the need to redeclare variable to inform what the result would be, like `isUserPreset(data.name);` also if isPresent property is not being used anywhere else you can just move checkUser() into if statement as is.

File: src/components/checkUser.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
 function checkUser(name) {
   const UserArray = [{ name: "User1" }, { name: "User2" }, { name: "User3" }];
-  const value = UserArray.find((val) => val.name === name);
-  if (value === undefined) {
-    return 0;
+  const value = UserArray.find((val) => val.name === name).name;
```
Comment (cq-ubaid-khan) [inline src/components/checkUser.js:3]: what if result of `UserArray.find((val) => val.name === name)` is undefined? accessing code after undefined in this case .name will throw exception.
Tip: remove .name after that and check for value in if statement as well => `if (value && value.name === name)`

File: src/app.js (hunk 1 above)
Comment (cq-mohtashim-ali) [reply src/app.js:36]: Done

File: src/components/checkUser.js (hunk 2 above)
Comment (cq-mohtashim-ali) [reply src/components/checkUser.js:3]: Done

**Pull request closed without merging**